pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "romanisim"
 description = "Nancy Grace Roman Space Telescope WFI Simulator"
 readme = "README.md"
-requires-python = ">=3.11"
+requires-python = ">=3.11,<3.14"
```

Comment:
By pinning python here we avoid the failure during 3.14 testing from the galsim dependency. If we go this route I'll open a follow-up PR that will release this pin (so we can re-trigger/update that PR as a way to check when galsim etc is updated). @schlafly I'm not familiar with galsim. Do you know if they have a release plan for 3.14?